setup.py

```diff
@@ -71,11 +71,7 @@ def run(self):
 
 # Extra dependencies for remote integrations
 
-# Restricting google-resumable-media version is due to the fact that
-# since 0.3.3 our binaries built by pyinstaller started to fail on Linux and
-# macOS. This restriction will be removed after
-# https://github.com/iterative/dvc/issues/2441 is resolved.
-gs = ["google-resumable-media==0.3.2", "google-cloud-storage==1.13.0"]
+gs = ["google-resumable-media>=0.4.0", "google-cloud-storage>=1.19.0"]
```

Comment:
I guess we no longer need to specify grm manually as long as we fix gcs.
```suggestion
gs = ["google-cloud-storage==1.19.0"]
```

Reply:
https://github.com/googleapis/google-cloud-python/blob/master/storage/setup.py#L34
```
    "google-resumable-media >= 0.4.0",
```
already specified in it's reqs.